lib/storage/src/content_manager/toc.rs

```diff
@@ -565,6 +569,9 @@ impl TableOfContent {
     }
 
     pub fn collection_wal_entry(&self, id: u64) -> raft::Result<RaftEntry> {
+        if id < 1 {
```

Comment:
why not checking if `id` is 0 instead?

Reply:
We can check that if you think it reads better. I just thought that as they mention they start the count from 1, it will read better like this